tcollector.py

```diff
@@ -727,6 +730,9 @@ def send_data(self):
                     protocol = 'http'
                 req = urllib2.Request('%s://%s:%s/api/put?details' % (
                     protocol, self.host, self.port))
+                if self.http_username and self.http_password:
+                  base64string = base64.encodestring('%s:%s' % (self.http_username, self.http_password)).replace('\n', '')
```

Comment:
Use `base64.b64encode()` instead of `base64.encodestring()`?

I was a bit confused as to what was going on here, and after running the code manually I noticed that `base64.encodestring()` adds a newline at the end, which was weird, but then after [RTFM'ing](https://docs.python.org/2/library/base64.html#base64.encodestring) I noticed the doc says:

> Encode the string `s`, which can contain arbitrary binary data, and return a string containing one or more lines of base64-encoded data. `encodestring()` returns a string containing one or more lines of base64-encoded data always including an extra trailing newline (`'\n'`).

and I saw that `b64encode()` seemed more appropriate.